src/test/java/teammates/test/cases/BaseComponentTestCase.java

```diff
@@ -31,6 +31,7 @@
 
     protected static final GaeSimulation gaeSimulation = GaeSimulation.inst();
     protected static final BackDoorLogic backDoorLogic = new BackDoorLogic();
+    protected DataBundle typicalBundle = getTypicalDataBundle();
```

Comment:
With this, the method body of `removeAndRestoreTypicalDataBundle` below can be simplified to one line: `removeAndRestoreDataBundle(typicalBundle)`.